mvcc/backend/read_tx.go

```diff
@@ -37,20 +37,49 @@ type ReadTx interface {
 	UnsafeForEach(bucketName []byte, visitor func(k, v []byte) error) error
 }
 
-type readTx struct {
-	// mu protects accesses to the txReadBuffer
-	mu  sync.RWMutex
+type baseReadTx struct {
```

Comment:
For better readability, could we add some comments explaining "baseReadTx" a base type for readTx and concurrentReadTx to avoid duplicated function implementation of `UnsafeForEach` and `UnsafeRange`?

Reply:
Done